typescript/sdk/src/gas/types.ts

```diff
@@ -14,6 +15,8 @@ export type IgpConfig = {
   owner: types.Address;
   beneficiary: types.Address;
   gasOracleType: ChainMap<GasOracleContractType>;
+  oracle: types.Address;
```

Comment:
Maybe `gasOracleOwner` or something similar? I interpreted this initially as an address that will be used as a gas oracle within the IGP